pkg/dohdec/types/doh.d.ts

```diff
@@ -136,6 +138,10 @@ export type DOH_LookupOptions = {
      * requires `json: false`.
      */
     dnssec?: boolean;
+    /**
+     * Disable DNSSec validation.
+     */
+    dnssecCd?: boolean;
```

Comment:
The rest of these options have more friendly names.  I wonder if we should spell out `dnssecCheckingDisabled`?